locales/ku_KU/duckduckgo.po

```diff
@@ -0,0 +1,5938 @@
+msgid "Bangs"
+msgstr "Bangs"
+
+#. Instant Answer tab name
+msgid "Barcode"
+msgstr "Barkod"
+
+msgctxt "settings"
+msgid "Basic"
+msgstr "Hêsan"
+
+msgctxt "theme"
+msgid "Basic"
+msgstr "Hêsan"
+
+msgid "Belgium (fr)"
+msgstr "Belçîka (fr)"
+
+msgid "Belgium (nl)"
+msgstr "Belçîka (nl)"
+
+#. This is an option for a user setting. It refers to showing the URL for a result BELOW the SNIPPET (description) of the result -- Example image: https://i.imgur.com/lnP77q4.png
+msgctxt "settingsvalue"
+msgid "Below Snippet"
+msgstr "Jêrî Danasînê"
+
+#. This is an option for a user setting. It refers to showing the URL for a result BELOW the TITLE of the result -- Example image: https://i.imgur.com/3cb6xx0.png
+msgctxt "settingsvalue"
+msgid "Below Title"
+msgstr "Jêrî Sernavê"
+
+msgctxt "settingsvalue"
+msgid "Best available"
+msgstr "Baştirîn berdest"
+
+#. In the right menu panel of the homepage.
+msgid "Beta"
+msgstr "Beta"
+
+msgctxt "video-duration"
+msgid "Between 4 and 20 minutes"
+msgstr "Di navbera 4 û 20 xulekan de"
+
+#. The color black.
+msgid "Black"
+msgstr "Reş"
+
+msgctxt "color"
+msgid "Black"
+msgstr "Reş"
+
+msgctxt "image-color"
+msgid "Black and white"
+msgstr "Reş û spî"
+
+msgid "Block Trackers"
+msgstr "Şopgeran asteng bike"
+
+msgctxt "reasons-to-use-duckduckgo"
+msgid "Block advertising trackers."
+msgstr "Şopgerên reklamê asteng bike."
+
+#. Part of a list of what our app can do. The app can: …
+msgctxt "welcome message eu search preference android"
+msgid "Block unsafe trackers"
+msgstr "Şopgerên neparastî asteng bike"
+
+#. shown after the DuckDuckGo extension is installed
+msgctxt "welcome message"
+msgid "Blocks trackers on websites you visit"
+msgstr "li ser malperên ku tu serdan dikî şopgeran asteng dike "
+
+msgid "Blog"
+msgstr "Blog"
+
+#. The color blue
+msgid "Blue"
+msgstr "Şîn"
+
+msgctxt "color"
+msgid "Blue"
+msgstr "Şîn"
+
+#. Click "Show Bookmarklet and Settings Data" - http://i.imgur.com/WyGptiV.png
+#. In the box that opens - http://i.imgur.com/tZAkHF6.png
+msgctxt "settings"
+msgid "Bookmarklet URL:"
+msgstr "Şûnpela URL:"
+
+#. The name of the Bosnian language
+msgctxt "language_name"
+msgid "Bosnian"
+msgstr "Boşnaqî"
+
+msgid "Brazil"
+msgstr "Brazîlya"
+
+msgctxt "color"
+msgid "Brown"
+msgstr "Qehweyî"
+
+msgid ""
+"Browse as usual while we add privacy protection. We bundled our search "
+"engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
+msgstr "Geranê xwe wekî her dem bike û em ê jî parastina taybetiyê tevlî bikin. Me motora lêgerînê xwe, astengkera şopgeran, û şîfrekirin di yek %s%sPêvek de pêk anî."
+
+msgid ""
+"Browse as usual, and we'll take care of the rest. We bundled our search "
+"engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
+msgstr "Geranê xwe wekî her dem bike, û em ê yên mayî li ser me ye. Me motora lêgerînê xwe, astengkera şopgeran, û şîfrekirin di yek %s%sPêvek de pêk anî."
```

Comment:
The source string has 3 placeholders (the `%s` symbol), but the translated string only has 2. This breaks our translation system, so please make sure to always match the number of placeholders. According to our tests there are other such instances, so please check the file using a specialised software like [Poedit](https://poedit.net/). It will highlight such inconsistencies for you. 🙂